src/Tribe/Linked_Posts/Base.php

```diff
@@ -339,7 +339,7 @@ public function find_with_upcoming_events( $only_with_upcoming = true, $event_po
 	 * @since TDB
 	 */
 	protected function prefix_key( $key ) {
-		if ( 0 !== strpos( $key, $this->meta_prefix ) && in_array( $key, Tribe__Events__Organizer::$meta_keys ) ) {
+		if ( 0 !== strpos( $key, $this->meta_prefix ) && in_array( $key, $this::$meta_keys, true ) ) {
```

Comment:
Strict PHP versions will flag this use of accessing a class property (`$meta_keys`) on the `$this` instance.

```suggestion
		if ( 0 !== strpos( $key, $this->meta_prefix ) && in_array( $key, static::$meta_keys, true ) ) {
```